webapp/src/sidepanel.tsx

```diff
@@ -32,6 +34,7 @@ interface SidepanelProps extends ISettingsProps {
     simSerialActive?: boolean;
     deviceSerialActive?: boolean;
     tutorialOptions?: pxt.tutorial.TutorialOptions;
+    /** An optional tutorial layout mode where the sim is on the left and instructions are at the top, regardless of screen size. */
```

Comment:
Is this comment regarding `tutorialSimSidebar`? Just wanted to check in since there isn't other code edited here.

Reply:
It is. I kept reading the code and going "What the heck is tutorialSimSidebar again?" so I added this. Formatting is weird, but that's because it's a doc comment so it appears nicely in intellisense:

![image](https://github.com/user-attachments/assets/62a0b2e7-424a-4e4a-9ced-8f1790f5e106)